api/v1alpha1/resource.proto

```diff
@@ -10,6 +10,7 @@ message Metadata {
     string type = 2;
     string id = 3;
     string version = 4;
+    string owner = 7;
```

Comment:
nit: out of order.

Reply:
I tried to preserve the tags to make sure it is backwards compatible, but I can break it if you're fine

Reply:
Oh I just meant that you can leave it as 7, but place it after six. But you do point out something I didn't consider. I guess the good thing about theses protobuf definitions for now are that we can change them with zero impact since it is completely self contained. In that case I would suggest "breaking things" even though in practice that doesn't mean much for us. Talos ships everything atomically.

Reply:
oh, this is not even consumed in Talos (yet), so doesn't matter, I was thinking about `os-engine`, but on other hand with other changes there's no backwards compatibility anyways, I'll get things updated